### PR TITLE
Search optimization for crawlability

### PR DESCRIPTION
**Pull request by ivarpruijn**
Several improvements for crawlability.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -5,14 +5,13 @@ const path = require('path');
...
-    + "are not equal");
+// only allow yarn start (dev) in dev mode
+if (nodeEnv === 'development' && objectivEnvironment !== 'development'){
+  throw new Error(`Not allowed to use 'yarn start' on non dev build (OBJECTIV_ENVIRONMENT=${process.env.OBJECTIV_ENVIRONMENT})`);
 }
-const isProductionEnv = objectivEnvironment ? objectivEnvironment.startsWith('prod') : false;
+const isProductionEnv = nodeEnv ? nodeEnv.startsWith('prod') : false;
 const isStagingEnv = objectivEnvironment ? (objectivEnvironment.startsWith('staging')) : false;
-const websiteUrl = isStagingEnv ? 'https://staging.objectiv.io/' : 'https://objectiv.io/';
+const websiteUrl = isStagingEnv ? 'https://staging.objectiv.io' : 'https://objectiv.io';
```
Comment (ivarpruijn) [inline docusaurus.config.js:14]: Trailing slash is not needed, and potentially messes up the meta og:url tag

File: docs/docs/tracking/core-concepts/locations.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ Tagging Sections can/should also be applied to pages/screens, see section
 :::
 
 ### Solving collisions
-See below for a simplified example taken from [our website's About page](https://objectiv.io/about), which 
+See below for a simplified example taken from [our website's About page](https://objectiv.io/about/), which
```
Comment (ivarpruijn) [inline docs/docs/tracking/core-concepts/locations.md:129]: Fixes a 301 for the crawler

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -123,11 +122,11 @@ const config = {
             items: [
               {
                 label: 'Privacy Policy',
-                to: websiteUrl + '/privacy'
+                to: websiteUrl + '/privacy/'
```
Comment (ivarpruijn) [inline docs/docusaurus.config.js:125]: Fixes a 301 for the crawler

File: docs/docusaurus.config.js [hunk 4]
```diff
@@ -123,11 +122,11 @@ const config = {
...
               },
               {
                 label: 'Cookies',
-                to: websiteUrl + '/privacy/cookies'
+                to: websiteUrl + '/privacy/cookies/'
```
Comment (ivarpruijn) [inline docs/docusaurus.config.js:129]: Fixes a 301 for the crawler

File: docs/docusaurus.config.js [hunk 5]
```diff
@@ -5,14 +5,13 @@ const path = require('path');
...
-    + "are not equal");
+// only allow yarn start (dev) in dev mode
+if (nodeEnv === 'development' && objectivEnvironment !== 'development'){
+  throw new Error(`Not allowed to use 'yarn start' on non dev build (OBJECTIV_ENVIRONMENT=${process.env.OBJECTIV_ENVIRONMENT})`);
 }
-const isProductionEnv = objectivEnvironment ? objectivEnvironment.startsWith('prod') : false;
+const isProductionEnv = nodeEnv ? nodeEnv.startsWith('prod') : false;
 const isStagingEnv = objectivEnvironment ? (objectivEnvironment.startsWith('staging')) : false;
-const websiteUrl = isStagingEnv ? 'https://staging.objectiv.io/' : 'https://objectiv.io/';
+const websiteUrl = isStagingEnv ? 'https://staging.objectiv.io' : 'https://objectiv.io';
```
Comment (ivarpruijn) [inline docs/docusaurus.config.js:14]: Trailing slash is not needed, and potentially messes up the meta og:url tag

**Review by borft — APPROVED** [review]
lgtm

**Pull request merged**